pinot-broker/src/main/java/org/apache/pinot/broker/routing/RoutingManager.java

```diff
@@ -324,6 +325,7 @@ public synchronized void buildRouting(String tableNameWithType) {
     // Add time boundary manager if both offline and real-time part exist for a hybrid table
     TimeBoundaryManager timeBoundaryManager = null;
     String rawTableName = TableNameBuilder.extractRawTableName(tableNameWithType);
+    Set<String> noReplicaSegments = new HashSet<>();
```

Comment:
Suggest changing to segmentsWithNoReplicas

Reply:
Updated.